tests/features/consumer/searchDocumentReference-success.feature

```diff
@@ -36,75 +36,42 @@ Feature: Consumer - searchDocumentReference - Success Scenarios
     And the organisation 'RX898' is authorised to access pointer types:
       | system                 | value     |
       | http://snomed.info/sct | 736253002 |
     And a DocumentReference resource exists with values:
-      | property    | value                                 |
-      | id          | 02V-1111111111-SearchMultipleRefTest1 |
-      | subject     | 9278693472                            |
-      | status      | current                               |
-      | type        | 736253002                             |
-      | category    | 734163000                             |
-      | contentType | application/pdf                       |
-      | url         | https://example.org/my-doc-1.pdf      |
-      | custodian   | 02V                                   |
-      | author      | 02V                                   |
-    And a DocumentReference resource exists with values:
-      | property    | value                                 |
-      | id          | 02V-1111111111-SearchMultipleRefTest2 |
-      | subject     | 9278693472                            |
-      | status      | current                               |
-      | type        | 736253002                             |
-      | category    | 734163000                             |
-      | contentType | application/pdf                       |
-      | url         | https://example.org/my-doc-2.pdf      |
-      | custodian   | 02V                                   |
-      | author      | 02V                                   |
-    And a DocumentReference resource exists with values:
-      | property    | value                                 |
-      | id          | 02V-1111111111-SearchMultipleRefTest3 |
-      | subject     | 9278693472                            |
-      | status      | current                               |
-      | type        | 887701000000100                       |
-      | category    | 734163000                             |
-      | contentType | application/pdf                       |
-      | url         | https://example.org/my-doc-3.pdf      |
-      | custodian   | 02V                                   |
-      | author      | 02V                                   |
+      | property    | value                           |
+      | id          | 02V-1111111111-SearchDocRefTest |
+      | subject     | 9278693472                      |
+      | status      | current                         |
+      | type        | 736253002                       |
+      | category    | 734163000                       |
+      | contentType | application/pdf                 |
+      | url         | https://example.org/my-doc.pdf  |
+      | custodian   | 02V                             |
+      | author      | 02V                             |
     When consumer 'RX898' searches for DocumentReferences with parameters:
       | parameter | value      |
       | subject   | 9278693472 |
+      | custodian | 02V        |
```

Comment:
As far as I can tell, tests a search using both NHS# + custodian filters. If there is only one pointer and it has a matching NHS# + custodian, in theory the scenario would pass even if either the subject filter or the custodian filter were not working. It might be good to additionally set up a few that we would expect to be filtered out: one with matching custodian but wrong NHS number, and vice versa.

Reply:
have added two new scenarios which isolate each filter:

Search for a DocumentReference by NHS Number and Custodian where NHS number filter matches one pointer from result set

Search for a DocumentReference by NHS Number and Custodian where Custodian filter matches one pointer from result set